bin/setup_ci

```diff
@@ -2,6 +2,15 @@
 
 set -e
 
+install_chromedriver() {
+  wget -N http://chromedriver.storage.googleapis.com/2.30/chromedriver_linux64.zip -P ~/
+  unzip ~/chromedriver_linux64.zip -d ~/
+  rm ~/chromedriver_linux64.zip
+  sudo mv -f ~/chromedriver /usr/local/share/
+  sudo chmod +x /usr/local/share/chromedriver
+  sudo ln -s /usr/local/share/chromedriver /usr/local/bin/chromedriver
```

Comment:
If you pass `-v` to `rm`, `mv`, `chmod`, and `ln`, you can see their output in the CI logs. Might be handy for debugging in the future.